socorro/mozilla_rulesets.py

```diff
@@ -91,14 +92,15 @@
     OutOfMemoryBinaryRule(),
     PHCRule(),
     BreadcrumbsRule(schema=get_schema("processed_crash.schema.yaml")),
-    JavaProcessRule(),
```

Comment:
"JavaProcessRule" isn't wildly helpful for what it does. It's a rule focusing on the `JavaStackTrace` annotation and processing that specifically. So I renamed it.